harvest/api/dummy.py

```diff
@@ -94,6 +95,9 @@ def fetch_latest_crypto_price(self) -> Dict[str, pd.DataFrame]:
 
     # -------------- Streamer methods -------------- #
 
+    def get_current_time(self):
+        return self.timestamp
```

Comment:
Override the base `get_current_time` to return the "current time" of the dummy trader